features/aanschrijfwijze.feature

```diff
@@ -147,72 +170,73 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
+      | V        | Francisca | van den     | Aedel         | JH              | Jonkvrouw F. van den Aedel | adellijke titel/predicaat wordt opgenomen in de vorm die hoort bij het geslacht van de persoon |
+      | M        | Franklin  | van den     | Aedel         | R               | F. ridder van den Aedel    |                                                                                                |
+      | V        | Francisca | van den     | Aedel         | R               | F. van den Aedel           | er is geen vrouwelijk vorm voor ridder                                                         |
+      | M        | Franklin  | van den     | Aedel         | B               | F. baron van den Aedel     |                                                                                                |
+      | V        | Francisca |             | Groenen       | BS              | F. barones Groenen         |                                                                                                |
+      | M        | Franklin  | van den     | Aedel         | H               | F. hertog van den Aedel    |                                                                                                |
+      | V        | Francisca |             | Groenen       | HI              | F. hertogin Groenen        |                                                                                                |
+      | M        | Franklin  | van den     | Aedel         | G               | F. graaf van den Aedel     |                                                                                                |
+      | V        | Francisca | van den     | Aedel         | GI              | F. gravin van den Aedel    |                                                                                                |
+      | M        | Franklin  | van den     | Aedel         | M               | F. markies van den Aedel   |                                                                                                |
+      | V        | Francisca | van den     | Aedel         | MI              | F. markiezin van den Aedel |                                                                                                |
+      | M        | Franklin  | van den     | Aedel         | P               | F. prins van den Aedel     |                                                                                                |
+      | V        | Francisca | van den     | Aedel         | PS              | F. prinses van den Aedel   |                                                                                                |
+      | M        | Franklin  | van den     | Aedel         | PS              | F. prins van den Aedel     |                                                                                                |
```

Comment:
Neem bij opmerkingen op:

'adellijke titel/predicaat wordt opgenomen in de vorm die hoort bij het geslacht van de persoon'